### PR TITLE
Add cluster-miq

### PR DESCRIPTION
**Pull request by pkliczewski**
Signed-off-by: Piotr Kliczewski <pkliczew@redhat.com>

### CODE REVIEW

**Comment by pkliczewski** [comment]
@aglitke @rmohr please review

File: README.md [hunk 1]
```diff
@@ -105,6 +105,43 @@ $ ./oc.sh get vms -o yaml | grep phase
...
+
+```bash
+./oc.sh get offlinevms
+No resources found.
+```
+
+In order to configure provider in `ManageIQ` there are two tokens needed by running
+
+```bash
+./oc sa get-token -n management-infra management-admin
```
Comment (rmohr) [inline README.md]: `./oc.sh`

File: README.md [hunk 2]
```diff
@@ -105,6 +105,43 @@ $ ./oc.sh get vms -o yaml | grep phase
...
+
+./oc.sh sa get-token -n default miq
```
Comment (rmohr) [inline README.md]: `./oc.sh`

File: Vagrantfile [hunk 3]
```diff
@@ -59,6 +59,13 @@ Vagrant.configure(2) do |config|
           "master" => ["master"],
         }
       end
+      master.vm.provision "miq", type: "ansible" do |ansible|
+        ansible.playbook = "miq.yaml"
+        ansible.groups = {
+          "nodes" => ["node"],
```
Comment (rmohr) [inline Vagrantfile]: nodes is not really needed here but it does not hurt either.

File: Makefile [hunk 4]
```diff
@@ -14,6 +14,9 @@ cluster-storage:
 cluster-kubevirt:
 	vagrant provision --provision-with=kubevirt
 
+cluster-miq:
```
Comment (rmohr) [inline Makefile:17]: Can you also add it to the all target?

**Review by rmohr — COMMENTED** [review]
Could you also add the ansible-playbook line in the README.md after the example inventory?

File: README.md (hunk 1 above)
Comment (pkliczewski) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (pkliczewski) [reply README.md]: Done

File: Vagrantfile (hunk 3 above)
Comment (pkliczewski) [reply Vagrantfile]: Will remove

File: Makefile (hunk 4 above)
Comment (pkliczewski) [reply Makefile:17]: Ack

**Review by aglitke — APPROVED** [review]

**Pull request merged**